### ISSUE TITLE
Visible screenshots

### ISSUE THREAD

**Issue by matisek2** [Snapcept/Snapcept#6]
not always, but I often get notifications about the screenshot and friends tell me about the screenshots

**Comment by FPSUsername** [comment]
v1.9.0 was the last 100% safe one, roll back to that one if you still have the apk file.

**Comment by matisek2** [comment]
Tomorrow I check this out

**Comment by tbrtje** [comment]
You cannot login to that version of Snapchat anymore. Is there an ETA on how long this needs to get fixed ?

**Comment by georgeravenholm** [comment]
@thies226j you sure that isnt because snapchat checks if the phone is rooted when you login?

**Comment by tbrtje** [comment]
@georgeravenholm yes I'm sure. The message I get when logging in tells me I am using an outdated version of Snapchat and asks me to update.

**Comment by georgeravenholm** [comment]
@thies226j heh
i make a titanium backup of snapchat so i can preserve the logged in state
did you get logged out automatically or did you like just install/log out manually?

**Comment by tbrtje** [comment]
@georgeravenholm that's what I do as well 😂.
But I get logged out automatically when trying this version.

**Comment by tbrtje** [comment]
When putting a .nomedia file in the snapcept directory, the screenshot notifications go away.

**Comment by georgeravenholm** [comment]
lmao luckily i already had one of those...
maybe it snapchat detects the images being added to the camera roll to try to detect this?

**Comment by georgeravenholm** [comment]
actually also maybe significant is that in the previous version, i sent myself a snap as a test and it was the very first snap downloaded and i got a screenshot notification for it and none since then, was it cause it just created all the folders or what
